src/System.Diagnostics.EventLog/src/System.Diagnostics.EventLog.csproj

```diff
@@ -54,6 +53,39 @@
     <Compile Include="$(CommonPath)\Interop\Windows\kernel32\Interop.WaitForSingleObject.cs">
       <Link>Common\Interop\Windows\kernel32\Interop.WaitForSingleObject.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)\CoreLib\Microsoft\Win32\SafeHandles\SafeLibraryHandle.cs">
+      <Link>Common\CoreLib\Microsoft\Win32\SafeHandles\SafeLibraryHandle.cs</Link>
```

Comment:
Where is the SafeLibraryHandle needed? (I am not able to find it by just looking at the code.)

Reply:
EventLogInternal.cs uses SafeLibraryHandle

Reply:
Ah ok. `SafeLibraryHandle.cs` is included above already. The duplicate definition should be deleted. Look for:

`<Compile Include="$(CommonPath)\CoreLib\Microsoft\Win32\SafeHandles\SafeLibraryHandle.cs">`